mass_mailing_resend/views/mass_mailing_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_mail_mass_mailing_form">
+        <field name="model">mail.mass_mailing</field>
+        <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button string="Resend"
```

Comment:
Don't you think "resend" is a bit misleading? Because you are not actually "resending" it, just setting back as draft. 🤔

Reply:
A `help`would also be useful here noting that the mail will only be sent to new recipients.

Reply:
I prefer resend word that is more user friendly, because setting back to draft is for resending purpose, although I'm going to add a text when clicked explaining about the new recipients.